eng/Version.Details.xml

```diff
@@ -3,23 +3,23 @@
   <ProductDependencies>
     <Dependency Name="Microsoft.WindowsDesktop.App.Runtime.win-x64" Version="3.1.2">
       <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>966115a91565d43cd545c1b8acd83eda2a0cb4cb</Sha>
+      <Sha>916b5cba268e1e1e803243004f4276cf40b2dda8</Sha>
```

Comment:
What does it mean that some Shas are updating but the versions aren't?  Is that because we have multiple builds with the same version numbers?  Should we be using unstable version numbers for these packages?

Reply:
ya, sha is the feed. We have multiple package with the same id and version